packages/sui-studio-utils/README.md

```diff
@@ -0,0 +1,146 @@
+# sui-studio-tools
+> A set of sui-studio usable tools.
+
+```sh
+$ npm install @s-ui/sui-studio-tools --save
+```
+
+# Domain builder
+
+> Domain builder have the purpose of give to you the possibility of mock some of non implemented domain use cases meanwhile your team are developing them.
```

Comment:
Domain builder *has* the purpose of *giving* you the possibility of *mocking*...